.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
-      - name: Use Xcode 14.3
+      - name: Use Xcode 15.1
         run: sudo xcode-select -switch /Applications/Xcode_14.3.app
```

Comment:
We'll also want to update the Xcode version in the `run` lines

Reply:
Weird, I thought I changed all of them. thanks!